configure.ac

```diff
@@ -228,6 +228,13 @@ if test "x$enable_expensive_hardening" = "xyes" || test "x$enable_fragile_harden
   AC_DEFINE(DEBUG_SMARTLIST, 1, [Enable smartlist debugging])
 fi
 
+AC_ARG_ENABLE(assert-fatalism,
+   AS_HELP_STRING(--enable-assert-fatalism, [force all soft asserts in Tor codebase (tor_assert_nonfatal(), BUG(), etc.) to act as hard asserts (tor_assert() and equivalents); makes Tor fragile and is not recommended for non-debug builds]))
```

Comment:
I'm a bit confused by the option name here.

"Fatalism" is the passive acceptance of the future, and might suggest that we are ignoring asserts.

Can we say "--enable-all-bugs-are-fatal" instead?

Reply:
Fixed.